docs/tutorials/kev-dev-with-skaffold.md

```diff
@@ -0,0 +1,97 @@
+---
+weight: 11
+title: Kev dev with skaffold
+---
+
+# Kev Dev w/Skaffold
+
+This tutorial will walk you through how to iterate on Kubernetes application using _Kev_ `dev` command with an optional [Skaffold](https://skaffold.dev/) hook.
+
+We encourage everyone to get familiar with [Getting started with Kev](getting-started-with-kev.md) tutorial before proceeding with this guide.
```

Comment:
`with [Getting started with Kev]...` -> `with the [Getting started with Kev]...`